### PR TITLE
Manual Request Editor Icon

### PR DESCRIPTION
**Pull request by kingthorin**
As discussed on issue #78.

- hand.png > The new icon.
- ManualRequestEditorDialog.java > Updated to set the new icon.
- ExtensionManualRequestEditor.java > Add public icon constant.
- ManualRequestEditorDialog.java > Add icon to context menu item.
- ExtensionStdMenus.java > Add icon to Tools menu item.

Ex: 
image

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: zap/src/main/java/org/parosproxy/paros/extension/manualrequest/ManualRequestEditorDialog.java [hunk 1]
```diff
@@ -70,6 +73,9 @@
     private static final long serialVersionUID = 1L;
 
     private static final Logger logger = Logger.getLogger(ManualRequestEditorDialog.class);
+    private static final Image MANUAL_REQUEST_ICON =
+            new ImageIcon(ManualRequestEditorDialog.class.getResource("/resource/icon/16/hand.png"))
```
Comment (thc202) [inline zap/src/main/java/org/parosproxy/paros/extension/manualrequest/ManualRequestEditorDialog.java]: Source/license?

**Comment by thc202** [comment]
The menu item `Tools` > `Manual Request Editor...` and context menu item `Open/Resend with Request Editor...` could be changed to have that icon.

File: zap/src/main/java/org/parosproxy/paros/extension/manualrequest/ManualRequestEditorDialog.java (hunk 1 above)
Comment (kingthorin) [reply zap/src/main/java/org/parosproxy/paros/extension/manualrequest/ManualRequestEditorDialog.java]: It’s a fugue icon

**Comment by kingthorin** [comment]
Okay, I’ll look at setting the menu icons too.

**Comment by kingthorin** [comment]
Menus updated.

**Comment by thc202** [comment]
Thanks!

**Pull request merged**